test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Great effort working on the README file as requested by the previous reviewer, 👏 👏  but I notice the `test.md` file issue raised by the reviewer has not been rectified yet. I can see you ignored it in the `.gitignore` file but it's still showing in your repo.
Kindly ignore it properly and be sure it's not showing again in your `repo` before you resubmit for review.
checkout this [article](https://www.freecodecamp.org/news/gitignore-file-how-to-ignore-files-and-folders-in-git/) for your peruse

Reply:
Thank you, I will work on it now.